compendium/ConfigurationAdmin/src/ConfigurationAdminImpl.cpp

```diff
@@ -493,8 +507,10 @@ void ConfigurationAdminImpl::RemoveConfigurations(
   auto idx = 0u;
   for (const auto& pidAndChangeCountAndID : pidsAndChangeCountsAndIDs) {
     const auto& pid = pidAndChangeCountAndID.pid;
-    if (removed[idx]) {
-      NotifyConfigurationUpdated(pid);
+    if (removedAndUpdated[idx].first) {
+      if (removedAndUpdated[idx].second) {
```

Comment:
minor: this can be combined into one `if` statement

Reply:
I don't think it can be. If removedAndUpdated[idx].first is true and removedAndUpdated[idx] is false we still have to log a debug message.

Reply:
I missed that. you're right.